docs/source/scripts_and_their_functions.rst

```diff
@@ -107,6 +107,17 @@ attributes are duplicated.
 For the default configuration of the node, see:
 ``ebu-run --admin.conf=ebu_tt_live/examples/config/deduplicator_fs.json``
 
+Denester Node
+-------------
+This node flattens nested ``div`` and ``span`` elements such that no
+``div`` ends up containing a ``div`` and no ``span`` ends up containing
+a ``span``. It also removes any ``p`` elements that specify a ``region``
+attribute that differs from a specified region on an ancester element.
+
+If nested ``div`` or ``p`` elements might be present in a document, the
```

Comment:
Should this say ``span`` instead of ``p``?

Reply:
Yes, thank you @PhoebeWheelerCode !